docs/Project.toml

```diff
@@ -1,5 +1,7 @@
 [deps]
 ADTypes = "47edcb42-4c32-4615-8424-f2b9edc5f35b"
+ColorSchemes = "35d6a980-a343-548e-a6ea-1d62b119f2f4"
```

Comment:
Add ColorTypes and ImageIO

Reply:
Images.jl is still required. ImagesIO is a markdown code block that isn't run, so it doesn't have to be added.